int64 source_group_index
int64 target_group_index
int64 source_cluster_id
bool do_slice
bool direction
---
bool success
